mysys/my_uuid_v7.c

```diff
@@ -0,0 +1,157 @@
+/* Copyright (c) 2024, MariaDB
```

Comment:
The CLA provides you joint copyright, you don't just have to list MariaDB (unspecified foundation/corporation)